spec/integration/rails_metrics_stress_spec.rb

```diff
@@ -28,7 +28,7 @@
 if enabled
   require 'action_controller/railtie'
 
-  RSpec.xdescribe 'Rails integration', :allow_running_agent, :spec_logger do
```

Comment:
lol what

Reply:
was that a typo or intentional?

Reply:
If a spec keeps being annoying, this is the treatment it can expect